Cookbook cleanup: Relocating 'Install on macOS or Windows' (#763)
ezsystems/developer-documentation: docs/community_resources/installing-on-mac-os-and-windows.md

# Install eZ Platform on macOS or Windows

> This page show how to perform a installation of eZ Platform on macOS or Windows.

Reviewer: ```suggestion
This page shows how to perform an installation of eZ Platform on macOS or Windows.
```

> This page show how to perform a installation of eZ Platform on macOS or Windows.

Reviewer: ```suggestion
This page explains how to install eZ Platform on macOS or Windows.
```

!!! caution

    This procedure is **for development purposes only**.
    Installing eZ Platform for production purposes is supported only on Linux.
    
    If you want to use eZ Platform in the production environment, see [Installing eZ Platform](../getting_started/install_ez_platform.md).  

### Prepare work environment

To install eZ Platform, you need a stack with MySQL and PHP.
Additionally, you need [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/lang/en/docs/install/) for asset management.
If you want to use a web server, you need to install it as well:
 
- For Windows: Apache
- For macOS: Apache/nginx

The instructions below assumes you are using Apache.

??? "Windows"

>     Locate `php.ini` file and open it in a text editor.

Reviewer: ```suggestion
    Locate the `php.ini` file and open it in a text editor.
```

Download and extract an archive into the location where you want your project root directory to be from [ezplatform.com](https://ezplatform.com/#download-option) (for open-source version) or from the [Support portal](https://support.ez.no/Downloads) (for eZ Enterprise), or clone the [GitHub repository](https://github.com/ezsystems/ezplatform):

``` bash

> git clone https://github.com/ezsystems/ezplatform

Reviewer: I think the full stop should remain here, to remain consistent with the main install guide.

    You can omit this step. If you do not create a database now, it will be created automatically in the next step.

Create a database by running the following command inside MySQL Shell:

``` bash
CREATE DATABASE ezplatform CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;
```

## Install eZ Platform

Before executing the following command, ensure that the user set during `composer install` has sufficient permissions.

> Install eZ Platform by running: `composer ezplatform-install`.

Reviewer: I'd leave the command in a separate code block. True, it is only one line, but it's one of the two main commands on this page and should be emphasised.
